StackOverflow/Controller/QuestionsTableViewController.m

```diff
@@ -0,0 +1,97 @@
+//
+//  QuestionsTableViewController.m
+//  StackOverflow
+//
+//  Created by Mali on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import "QuestionsTableViewController.h"
+#import "QuestionsCell.h"
+#import "DataFetchDelegate.h"
+#import "QuestionsDataObject.h"
+#import "QuestionsData.h"
+
+
+@interface QuestionsTableViewController () <DataFetchDelegate> {
+    NSArray *dataResult;
```

Comment:
This is neither data nor a result. Variables should be named after their role not their type. What role does this array fulfil ? Is it a list of something, then call it that. Calling something dataResult is meaningless. Also please be explicit when defining the storage modifiers of variables, in this case I assume you want this variable to have a strong reference ( which is the default ) therefore annotate this with __strong so that other developers know that is what you mean. ALWAYS be explicit when annotating variables and properties.